src/Navigation.tsx

```diff
@@ -464,19 +470,25 @@ const LINKING = {
 
 function RoutesContainer({children}: React.PropsWithChildren<{}>) {
   const theme = useColorSchemeStyle(DefaultTheme, DarkTheme)
+  const {currentAccount} = useSession()
+  const {openModal} = useModalControls()
+
+  function onReady() {
```

Comment:
Verified this re-fires when we change the account — due to the key thing at the top.

I.e. logging out and in (barring the suppression condition for the storage) shows the modal.